### ISSUE TITLE
Translating the Data Hazards

### ISSUE THREAD

**Issue by ninadicara** [very-good-science/data-hazards#193]
As mentioned by @vairylein it would be very nice to have translations of the Data Hazards available, and we may have some people willing to do that for us. 

It would be useful to think about 
- [ ] How this could be displayed on the website within the bounds of our current Sphinx setup (maybe using tabs?)
- [ ] How we make sure this is kept up to date, or at least make sure we properly document the corresponding versions.
- [ ] Make sure we are totally happy with the English language versions before translating them.

**Comment by ninadicara** [comment]
Note - have a look at how the Turing Way does this, they must have some similar problems but I know they do have translations

**Comment by ninadicara** [comment]
Okay - my thought is that on the Data Hazards page we can have links to 'View this page in German' etc at the top. This would port you to `datahazards.com/etikett` (for german, for instance) and all the Hazard label pages from this root page would be in this language too. We'd achieve this just by adding the new pages to a folder called `etikett` and adding an `ettiket.md` page to the root folder so the URLS still work. This is a fairly simple but hopefully effective way to achieve this!

**Comment by vairylein** [comment]
Yes! I agree. we should focus on translating the hazard labels first. Maybe there is a way of condensing the basic instructions and project ethos to a one-pager that we could have available in all the languages without the crazy overhead of keeping absolutely everything up to date? I am in touch with a German data institute that would be interested in supporting the German version (through funding or a translator) and I would be happy to supervise it. I'll keep you up to date if that goes through!

**Comment by ninadicara** [comment]
Love the idea of a one pager! That would work perfectly :)
